rologaki.ino

```diff
@@ -112,7 +114,7 @@ void loop()
     }
   }
     t=millis();
-    if(t>=x+2000){
+    if(t>=x+2000){ // TODO: why 2000?
```

Comment:
why 2000? explain. also don't hardcode this. Put it as constant on top of the file.